OpenRA.Game/Scripting/ScriptContext.cs

```diff
@@ -201,7 +201,7 @@ public sealed class ScriptContext : IDisposable
 			using (var loadScript = (LuaFunction)runtime.Globals["ExecuteSandboxedScript"])
 			{
 				foreach (var s in scripts)
-					loadScript.Call(s, world.Map.Open(s).ReadAllText()).Dispose();
+					loadScript.Call(s, ((Map)world.Map).Open(s).ReadAllText()).Dispose();
```

Comment:
if its not high performance code, i would try a cast. this way future mods can keep using default functionality.

Reply:
Don't understand, the above is casting to `Map`. If you mean cast to `IMap`, this would be difficult as I would have to change the Lua handling part of the code base. I don't think this is needed for the purpose of this PR, which is primarily to introduce map formats and abstract away some main functions of Maps into interfaces. Once again it could be looked at in a future PR.